packages/server/lib/controllers/proxy.coffee

```diff
@@ -75,6 +75,20 @@ module.exports = {
 
         return res.status(503).end()
 
+    ## if we have white listed hosts
+    if wlh = config.whitelistHosts
+      ## and url does not match any of our whitelisted hosts
+      if not matched = blacklist.matches(req.proxiedUrl, wlh)
```

Comment:
I am confused by the comment and whitelist / blacklist combination here

Reply:
The whitelist is being passed in to this function as `wlh`, this is just a (now badly named) function. `blacklist` (the library) is now more a `*list`-matching library after these changes so presumably should be renamed.